### ISSUE TITLE
Android building errors

### ISSUE THREAD

**Issue by noaht8um** [badaix/snapcast#192]
I am having issues building the android app from source. I am using the develop branch and followed the guide for Android but it's failing on the final ./build_android.sh script. At first it couldn't find the FLAC directory so I had to symlink it into the client directory. That got it farther, but now I'm getting several Flac errors. I hate to dump a bunch of logs, but I'll attach what I got when I ran the script.
[dump.txt](https://github.com/badaix/snapcast/files/802749/dump.txt)

**Comment by badaix** [comment]
did you build the externals?
```
$ cd <snapcast dir>/externals
$ ./build_externals_android.sh
```

**Comment by noaht8um** [comment]
@badaix I did, yes and they compiled fine. I'm going to try again now to double-check though.

**Comment by noaht8um** [comment]
I didn't notice this yesterday, but when I run the build_externals_android.sh script, this is what I get at the very beginning:
`Missing program 'libtool'.`
 
The script ran anyway, but I think that was the problem. You may want to add libtool-bin to the list of dependencies for compiling the Android app. 

It seemed to work fine. My follow up question is how do you compile the actual APK for android? Is it a typical "/usr/local/bin/gradle assemble" command? If so, I'd add that to the Android guide as well.

**Comment by noaht8um** [comment]
Ah, I just Googled it and realized you just have to run the "gradlew" script. 
I ran it and it required the android sdk to be defined either in the local.properties file or with the ANDROID_HOME environment variable. Might be worth adding to the guide.
I used export ANDROID_HOME="/path/to/android/home" and it worked fine.
After that, it threw several Android SDK license errors, so I'll get back when I fix that.
